File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/TreeView/Features/StateImagePage.vb
Namespace CompanionKit.Controls.TreeView.Features

	Public Class StateImagePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

            ' Set the image list that is used to indicate the state of the TreeView and its nodes.
            Me.mobjTreeView.StateImageList = New ImageList
            Me.mobjTreeView.StateImageList.Images.Add(New ImageResourceHandle("16x16.DefaultTreeNode.gif"))
            Me.mobjTreeView.StateImageList.Images.Add(New ImageResourceHandle("16x16.SelectedTreeNode.gif"))

		End Sub
	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/TreeView/Features/StateImagePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.TreeView.Features
{
    public partial class StateImagePage : UserControl
    {
        public StateImagePage()
        {
            InitializeComponent();

            // Set the image list that is used to indicate the state of the TreeView and its nodes.
            this.mobjTreeView.StateImageList = new ImageList();
            this.mobjTreeView.StateImageList.Images.Add(new global::Gizmox.WebGUI.Common.Resources.ImageResourceHandle("16x16.DefaultTreeNode.gif"));
            this.mobjTreeView.StateImageList.Images.Add(new global::Gizmox.WebGUI.Common.Resources.ImageResourceHandle("16x16.SelectedTreeNode.gif"));
        }
    }
}